Frontend/ui-driver/src/Helpers/Utils.js

```diff
@@ -861,4 +862,18 @@ export const istToUtcDate = function (dateStr) {
 
 export const setValueToLocalStore = function (key,value){
   JBridge.setInSharedPrefs(key, value);
+}
+
+export const setPopupType = function (popupType) {
+  this.popupType = popupType;
+};
+
+export const getPopupType = function (just, nothing) {
```

Comment:
where and for what purpose is this function for?

Reply:
This is to retain the GoToTimeExpired popup state